libs/category/testing/src/inmemory-backend/category.service.ts

```diff
@@ -77,13 +79,14 @@ export class DaffInMemoryBackendCategoryService implements InMemoryDbService {
 
 	private generateProductIdArray(products: DaffProduct[]) {
```

Comment:
Can some of this randomization logic be extracted? It took a while for me to understand what precisely was going on in this method.

Reply:
put in core